delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/StoragePluginModule.scala

```diff
@@ -94,6 +95,22 @@ class StoragePluginModule(priority: Int) extends ModuleDef {
         )
     }
 
+  make[StoragePermissionProvider].from { (storages: Storages) =>
+    new StoragePermissionProvider {
+      override def permissionFor(id: IdSegmentRef, project: ProjectRef, read: Boolean): UIO[Permission] =
+        storages
+          .fetch(id, project)
+          .map(storage => storage.value.storageValue)
+          .map(storage =>
+            read match {
+              case true  => storage.readPermission
+              case false => storage.writePermission
+            }
+          )
+          .hideErrorsWith(_ => new RuntimeException("bob"))
```

Comment:
This error needs fixing

Reply:
In the CE migration we extends different rejections with the `Rejection` trait. Doing so for the fetch rejection would allow to simply use `.hideErrors` here